Sources/Redis/Application+Redis.swift

```diff
@@ -0,0 +1,72 @@
+import Vapor
+
+extension Application {
+    public var redis: Redis {
+        .init(application: self)
+    }
+
+    public struct Redis {
+        struct ConfigurationKey: StorageKey {
+            typealias Value = RedisConfiguration
+        }
+
+        public var configuration: RedisConfiguration? {
+            get {
+                self.application.storage[ConfigurationKey.self]
+            }
+            nonmutating set {
+                self.application.storage[ConfigurationKey.self] = newValue
+            }
+        }
+
+
+        struct PoolKey: StorageKey, LockKey {
+            typealias Value = EventLoopGroupConnectionPool<RedisConnectionSource>
+        }
+
+        internal var pool: EventLoopGroupConnectionPool<RedisConnectionSource> {
+            if let existing = self.application.storage[PoolKey.self] {
+                return existing
+            } else {
+                let lock = self.application.locks.lock(for: PoolKey.self)
+                lock.lock()
+                defer { lock.unlock() }
+                guard let configuration = self.configuration else {
+                    fatalError("Redis not configured. Use app.redis.configuration = ...")
+                }
+                let new = EventLoopGroupConnectionPool(
+                    source: RedisConnectionSource(configuration: configuration, logger: self.application.logger),
```

Comment:
Not sure about reusing the Application logger here, might be preferable to use the logger provided by redis-kit so that users can override it in their Logging Bootstrap.

Reply:
All loggers will be overridden by `LoggingBootstrap`, including application's. I think application's logger makes sense here since this is a property of application.